index.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio</title>
+        <meta name="description" content="Portfolio">
+        <meta name="author" content="Sanja">
+        <link rel="stylesheet" href="style.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    </head>
+
+    <body>
+        <section id="base">
+            <header>
+                <div id="topHead">
+                    <p id="time">9:41</p>
+                    <ul id="appBar">
+                        <li class="app"><img src="Icons/CellularConnection.png" alt="CellularConnection Icon" width="18" height="11"></li>
+                        <li class="app"><img src="Icons/Wifi.png" alt="WiFi Icon" width="15" height="11"></li>
+                        <li class="app"><img src="Icons/Battery.png" alt="Battery Icon" width="18" height="8"></li>
+                    </ul>
+                    
+                </div>
+                <div id="bottomHead">
+                    <h3 id="logo">Sanja Mandic</h3>
```

Comment:
- [ ] I kindly suggest you surround you logo with an `a tag` because users can click on your logo to easily navigate back to the homepage